src/components/DataDecorator/_mixins.scss

```diff
@@ -47,8 +47,11 @@
   padding: unset;
   border: unset;
   vertical-align: middle;
+  color: $text-02;
+
+  &#{$root}--interactive {
```

Comment:
Is there a reason that you need to increase the specificity of this selector?

Reply:
Because I don't want to rely on style order to set a text color 👍

Reply:
Based on the changes I'm seeing, the specificity can go down and it should apply the styles as expected. Higher specificities are hard to work with as a consumer.

Reply:
To illustrate, this relies on order:

```
.security--decorator {
  color: $text-02;
}

.security--decorator--interactive {
  color: $hover-primary-text;
}
```

And this relies on specificity:

```
.security--decorator {
  color: $text-02;
}

.security--decorator.security--decorator--interactive {
  color: $hover-primary-text;
}
```

We should lean on adding specificity when necessary, instead of relying on style order, to get desired results.

In fact there is currently a situation where `.bx--link` color styles are overriding the `.security--decorator` color rule, due solely to style order, if you  use the browser inspector and check out which text color is being applied, BEFORE I override it with `.security--decorator.security--decorator--interactive`